pentasharp/Models/DTOs/TaxiCompanyDto.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace pentasharp.Models.DTOs
+{
+    /// <summary>
+    /// Represents a DTO for a taxi company, including details about the company.
+    /// </summary>
+    public class TaxiCompanyDto
```

Comment:
dont use Dto in namings, remove dto here and every other place you're using it. Also, you dont need the DTO folder, this is a model that can be fit in the model folder. If you want to be more specific, you can create a TaxiRequests folder and fit there TaxiCompanyRequest, TaxiRequest, TaxiReservationRequest or whatever you rename these.